README.md

```diff
@@ -1 +1,50 @@
-# rawhide-rpmdeplint-pipeline
+# rpmdeplint-pipeline
+
+This repository contains definition of the rpmdeplint CI pipeline.
```

Comment:
Another small one:

"This repository contains the definition of the rpmdeplint CI pipeline."